controllers/pullRequestsController.js

```diff
@@ -43,6 +43,52 @@ const getPRdetails = async (req, res) => {
   }
 }
 
+const getStalePRs = async (req, res) => {
```

Comment:
Please add JSDoc.

Reply:
Added in the review-related changes. Thanks!